api_v2/src/overtime/overtime.resolver.ts

```diff
@@ -16,4 +17,16 @@ export class OvertimeResolver {
   async overtime(@Args('userId') userId: number) {
     return await this.overtimeService.getOvertimeByUser(userId);
   }
+
+  @Mutation(() => String)
+  async approveDisapproveAllOvertimeSummary(
+    @Args('approvalData') approvalData: ApproveOvertimeSummaryRequestInput,
+  ): Promise<string> {
+    try {
+      await this.overtimeService.approveDisapproveAllOvertimeSummary(approvalData);
+      return SuccessMessageEnum.OVERTIME_SUMMARY_REVIEW_SUBMITTED;
+    } catch (error) {
+      return `Error: ${error.message || 'An unexpected error occurred.'}`;
```

Comment:
You can handle the error like this, Returning error messages as normal responses is bad practice.
```suggestion
 throw new InternalServerErrorException(error.message || 'An unexpected error occurred.');
```

Reply:
@Mjdelima20 done sir